docs/usage/self-hosted-experimental.md

```diff
@@ -100,6 +100,14 @@ Allowed values:
 
 Default value: `asc`.
 
+## `RENOVATE_X_REBASE_PAGINATION_LINKS`
+
+<!-- prettier-ignore -->
+!!! note
+    For the GitHub Enterprise Server platform only.
+
+If set, Renovate will rewrite GitHub Enterprise Server's pagination responses to use the base URL from the Renovate config.
```

Comment:
```suggestion
If set, Renovate will rewrite GitHub Enterprise Server's pagination responses to use the endpoint URL from the Renovate config.
```

@HonkingGoose should we use endpoint here, i think that's the name of the config option.

Reply:
I don't know enough about this to answer your question. 😉 Here's the docs for the `endpoint` config option (self-hosted only!):

https://docs.renovatebot.com/self-hosted-configuration/#endpoint

Reply:
`endpoint` should always point to the platform renovate runs against. so it should always be set even on hosted app, because it defaults to `api.github.com` there

Reply:
```suggestion
If set, Renovate will rewrite GitHub Enterprise Server's pagination responses to use the `endpoint` URL from the Renovate config.
```